internal/cli/analytics_test.go

```diff
@@ -26,6 +26,7 @@ func TestNormalizedGitRemoteUsername(t *testing.T) {
 }
 
 func TestGitOrigin(t *testing.T) {
+	t.Fatal("foo")
```

Comment:
👀

Reply:
forcing a failure to test the behavior before I add reviewers

AFAIK there's no way to test this without creating a PR

Reply:
ahhh I gotcha